playgrounds/petclinic-maven-sb-2/src/main/scss/header.scss

```
@@ -0,0 +1,73 @@
+.navbar li:hover > a {
+  color: #eeeeee;
+  background-color: #6db33f;
+}
+
+.navbar-toggle {
+  border-width: 0;
+
+  .icon-bar + .icon-bar {
+    margin-top: 3px;
+  }
+  .icon-bar {
```

Stylelint (17.14.0)

[error] Expected selector ".icon-bar" (".navbar-toggle .icon-bar") to come before selector ".icon-bar + .icon-bar" (".navbar-toggle .icon-bar + .icon-bar"), at line 65 (no-descending-specificity)

(no-descending-specificity)